html_xblock/html.py

```diff
@@ -0,0 +1,117 @@
+"""This XBlock will help creating a secure and easy-to-use HTML blocks in edx-platform."""
+
+import logging
+import pkg_resources
+from xblock.core import XBlock
+from xblock.fields import Scope, String
+
+from xblock.fragment import Fragment
+
+from .bleaching import SanitizedText
+from .utils import _
+
+log = logging.getLogger('XBlock.HTML')  # pylint: disable=invalid-name
+
+
+class HTML5XBlock(XBlock):
+    """
+    This XBlock will provide an HTML WYSIWYG interface in Studio to be rendered in LMS.
+    """
+    display_name = String(
+        display_name=_('Display Name'),
+        help=_('The display name for this component.'),
+        scope=Scope.settings,
+        default=_('Text')
+    )
+    data = String(help=_('Html contents to display for this module'), default=u'', scope=Scope.content)
+    editor = String(
+        help=_(
+            'Select Visual to enter content and have the editor automatically create the HTML. Select Raw to edit '
+            'HTML directly. If you change this setting, you must save the component and then re-open it for editing.'
+        ),
+        display_name=_('Editor'),
+        default='visual',
+        values=[
+            {'display_name': _('Visual'), 'value': 'visual'},
+            {'display_name': _('Raw'), 'value': 'raw'}
+        ],
+        scope=Scope.settings
+    )
+
+    @staticmethod
+    def resource_string(path):
+        """Handy helper for getting resources from our kit."""
+        data = pkg_resources.resource_string(__name__, path)
+        return data.decode('utf8')
+
+    @XBlock.supports('multi_device')
+    def student_view(self, context=None):  # pylint: disable=unused-argument
+        """
+        Return a fragment that contains the html for the student view
+        """
+        html = self.resource_string('static/html/lms.html')
+        frag = Fragment(html.format(self=self))
+        frag.add_css(self.resource_string('public/plugins/codesample/css/prism.css'))
+        frag.add_javascript(self.resource_string('public/plugins/codesample/js/prism.js'))
+        frag.initialize_js('HTMLXBlock')
```

Comment:
There is no JavaScript file loaded in the `student_view` so this is causing an error in the JS console:

```
xblock.js:67 TypeError: Cannot read property 'prototype' of undefined
```

I think you can just remove this until unless/until we need JavaScript code to run the `student_view`.